### PR TITLE
Slight bits more readable

### PR DESCRIPTION
**Pull request by blaggacao**

### CODE REVIEW

**Comment by blaggacao** [comment]
@jreese If you'd agree (I have the right use cases in mind), I'd fix the test.

File: bowler/helpers.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ def print_selector_pattern(
             for child in node.children:
                 print_selector_pattern(child, results, filename)
             click.echo("> ", nl=False)
+    click.echo("\n", nl=False)
```
Comment (jreese) [inline bowler/helpers.py]: Why not just `click.echo()`?

Comment (blaggacao) [reply bowler/helpers.py]: No particular reason, I just wanted to keep with the `nl=False` style in the surroundings.
Should I change?

**Comment by thatch** [comment]
Rebased to trigger modern CI.  @jreese any objections?

**Comment by jreese** [comment]
Don't know why CI is failing so hard on this, but it looks fine to me. If tests pass locally with your rebase, just merge it.

**Comment by thatch** [comment]
Rebased again and CI is happy now.

**Pull request merged**